skimage/transform/radon_transform.py

```diff
@@ -216,26 +216,37 @@ def iradon(radon_image, theta=None, output_size=None,
     img = np.pad(radon_image, pad_width, mode='constant', constant_values=0)
 
     # Construct the Fourier filter
-    f = fftfreq(projection_size_padded).reshape(-1, 1)   # digital frequency
-    omega = 2 * np.pi * f                                # angular frequency
-    fourier_filter = 2 * np.abs(f)                       # ramp filter
+    n1 = np.arange(0, projection_size_padded / 2 + 1, dtype=np.int)
+    n2 = np.arange(projection_size_padded / 2 - 1, 0, -1, dtype=np.int)
+    n = np.concatenate((n1, n2))
+    f = np.zeros(projection_size_padded)
+    f[0] = 0.25
+    f[1::2] = -1 / (np.pi * n[1::2])**2
+
+    omega = 2 * np.pi * fftfreq(projection_size_padded)
+    fourier_filter = 2 * np.real(fft(f))         # ramp filter
     if filter == "ramp":
         pass
     elif filter == "shepp-logan":
         # Start from first element to avoid divide by zero
-        fourier_filter[1:] = fourier_filter[1:] * np.sin(omega[1:]) / omega[1:]
+        fourier_filter[1:] *= np.sin(omega[1:] / 2) / (omega[1:] / 2)
```

Comment:
Why did we need to divide by 2 here again?

Reply:
![image](https://user-images.githubusercontent.com/90008/53691378-dad86600-3d4a-11e9-8233-a894188d48f7.png)
Left is the original fourier_filter at the endof this line, Right is the one you are proposing. Pretty drastically different. is this something you meant to change?

Reply:
@hmaarrfk I believe that change was one of the main points of the PR. See #2895

Reply:
@philipstarkey @kczimm you are both right..... yeah, looking at this PR again after so long introduces makes merging more difficult. It seems there is a small artifact at point 33. I think it is from the way we are suggesting to compute the ramp filter.

![image](https://user-images.githubusercontent.com/90008/53695934-96bb8480-3d8f-11e9-9298-7707fd0d8c22.png)

Reply:
This is probably the desired artifact...

Reply:
what parameters are you using to create this filter? I would like to reproduce it.